variables.tf

```diff
@@ -0,0 +1,37 @@
+
+
+variable "location" {
+description = "The Azure location where all resources in this deployment should be created"
+default = "uksouth"
+}
+
+variable "GITHUBID" {
+description = "The githubid"
+default = "47190936"
+}
+
+variable "WEBAPPLICATIONCLIENT" {
+description = "client web"
+default = "88de9a9a5377861febef"
+}
+variable "CONNECTIONSTRING" {
+description = "mongo connection"
+default = "mongodb://tododatabaseproject:JAyasz79f7yRR2zqi34xJG5PkjXbaRlvKJC4XlUYrqezapD97bHUF9WFeIpoDxXdaUNhby7ohYnSYK5Pommsfw==@tododatabaseproject.mongo.cosmos.azure.com:10255/?ssl=true&replicaSet=globaldb&retrywrites=false&maxIdleTimeMS=120000&appName=@tododatabaseproject@"
```

Comment:
Note that this has your database password in, we should definitely:
* Remove this from the variables.tf file!
* Change your DB password in Azure so that this can no longer connect successfully

Once we've done that, this & some of these other input values are sensitive, so [we should mark them](https://www.terraform.io/docs/language/values/variables.html#suppressing-values-in-cli-output) as such to prevent them being logged to the console